### ISSUE TITLE
How to ignore errors

### ISSUE THREAD

**Issue by akashnimare** [getsentry/sentry-electron#117]
I'm having a hard time ignoring the errors. Can somebody tell me the correct configuration to ignore particular error cc @HazAT.

**Comment by akashnimare** [comment]
Hmm, this is what I'm using -
```js

const { init, captureException } = require('@sentry/electron');
const isDev = require('electron-is-dev');

const sentryInit = () => {
	if (!isDev) {
		init({
			dsn: 'DSNHERE',
			ignoreErrors: ['Network failed']
		});
	}
};

module.exports = {
	sentryInit,
	captureException
};

```

Let me see if this works. Also, is there any way to disable the complete console logs here like I don't want to send the console log to the Sentry.

**Comment by TuxujPes** [comment]
@akashnimare 

```
autoBreadcrumbs: {
    'console': false
}
```

https://docs.sentry.io/clients/javascript/config/

**Comment by timfish** [comment]
We have just released v3.0.0 of the SDK which includes a lot of changes and improvements since this issue was opened. 

If you can still reproduce this with v3, please open a new issue and I'll look into it.
